src/components/tx/SignOrExecuteForm/PermissionsCheck/__test__/PermissionsCheck.test.tsx

```diff
@@ -18,6 +18,17 @@ import { FEATURES } from '@/utils/chains'
 import { chainBuilder } from '@/tests/builders/chains'
 import { useHasFeature } from '@/hooks/useChains'
 
+// Mock fetch
+Object.defineProperty(window, 'fetch', {
+  writable: true,
+  value: jest.fn(() =>
+    Promise.resolve({
+      ok: false,
+      json: () => Promise.resolve({}),
+    }),
+  ),
+})
```

Comment:
Do we know which call ends up doing the fetch? Imo we shouldn't mock fetch but instead mock our own functions that wrap the fetch calls. It seems we are overwriting the global object, do you know if this is cleaned up after the tests run?

Reply:
We use the same type of mock in the GasPrice tests and it seems to be fine.